quickstart.md

```diff
@@ -10,30 +10,25 @@ on [Pre-Setup](/pre-setup.html)
 
 ## Installation
 
-1. Clone the CitrineOS repository to your local machine:
+**1. Clone the `citrineos/core` repository onto your local machine:**
 
-    ```shell
     git clone https://github.com/citrineos/citrineos-core
-    ```
 
-1. Navigate to the `citrineos-core/Server` directory and run the entire required stack with docker-compose.
+To include OCPI, additionally clone the `citrineos/ocpi` repository:
+
+    git clone https://github.com/citrineos/citrineos-ocpi
+
+**2. Navigate to the `citrineos-core/Server` directory:**
 
-    ```shell
     cd citrineos-core/Server
-    docker-compose up -d 
-    ```
-
-    The expected outcome should look like this:
```

Comment:
Probably a controversial change, but I am not really sure it is helpful to include this expected output as it seems trivial and only adds more clutter to what should be a concise as possible quick start.